pom.xml

```diff
@@ -36,7 +36,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.product</groupId>
       <artifactId>samplesvc-api</artifactId>
-      <version>10.49.6</version>
+      <version>10.49.7-SNAPSHOT</version>
```

Comment:
Will need to change this to release version once https://github.com/ONSdigital/rm-samplesvc-api/pull/6 is merged in.

Reply:
:+1: